src/disclosure-manager/_disclosureCallbacks.js

```diff
@@ -0,0 +1,26 @@
+let disclosureCallbacks = [];
+
+const addCallback = (callback) => {
+  disclosureCallbacks = [...disclosureCallbacks, callback];
+};
+
+const removeCallback = (callback) => {
+  disclosureCallbacks = disclosureCallbacks.filter(disclosureCallback => disclosureCallback !== callback);
+};
+
+const closeMostRecentDisclosure = () => {
+  if (!disclosureCallbacks.length) {
+    return Promise.resolve();
+  }
+
+  return disclosureCallbacks[disclosureCallbacks.length - 1]();
+};
+
+const getActiveDisclosureCount = () => (disclosureCallbacks.length);
```

Comment:
Out of curiosity, how or when do we expect this to be used?

Reply:
When an event is triggered (for example, the android back button) the consumer can call this to help determine what action to take.  If there are disclosures, they would probably call closeMostRecentDisclosure.  If not, they would take some other action.

Reply:
can we add some document on the use case?

Reply:
Originally I had some documentation, but we decided to remove it (conversation is hidden above) since this isn't really intended to be used externally.

Reply:
@tbiethman against in-line docs for developers?

Reply:
I'm not opposed to inline docs. I just really don't want consumers getting ahold of this and using it in their workflows.

Reply:
Added some inline doc for the publicly exported functions: af5f40475a7c554a667353d1765d7b8c0956dba4